pages/[type]/[id].vue

```diff
@@ -427,19 +427,15 @@
           :auth="auth"
           :tags="tags"
         />
-        <div v-else-if="project.status === 'withheld'" class="card warning" aria-label="Warning">
+        <MessageBanner v-if="project.status === 'withheld'" message-type="warning">
```

Comment:
Keep this one as an else-if, it shouldn't be displayed to members of the project